### DISCUSSION TITLE
整合包崩了

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#2496] [提问] [answered]
[错误报告-2023-8-14_21.56.17.zip](https://github.com/Hex-Dragon/PCL2/files/12335875/-2023-8-14_21.56.17.zip)
未知原因，一启动就崩溃，重装后哪怕不做改动也没用
Antimatter Chemistry1.4.4最新版

**Comment by zkitefly** [comment, marked as answer]
下载这个配置文件 [https://raw.githubusercontent.com/dogboy21/serializationisbad/master/serializationisbad.json](https://gp.zkitefly.eu.org/https://raw.githubusercontent.com/dogboy21/serializationisbad/master/serializationisbad.json)

然后将刚刚下载的文件覆盖到 `D:\Desktop\Empty\default\minecraft s\We don't know\.minecraft\versions\AC1.4.4\config\serializationisbad.json`

**Reply by iPodtouch6557** [reply]
感谢了，问题解决
原因竟是电脑里都没有这个文件……

**Reply by hejiehao** [reply]
方案来源：HMCL KOOK
